pages/common/ffmpeg.md

```diff
@@ -14,10 +14,18 @@
 
 `ffmpeg -i {{frame_%d.jpg}} -f image2 {{video.mpg|video.gif}}`
 
-- Extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
+- Quickly extract a single frame from a video at time mm:ss and save it as a 128x128 resolution image:
 
-`ffmpeg -i {{video.mp4}} -ss {{mm:ss}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
+`ffmpeg -ss {{mm:ss}} -i {{video.mp4}} -frames 1 -s {{128x128}} -f image2 {{image.png}}`
 
-- Convert AVI video to MP4. AAC Audio @ 128kbit, Video @ 1250Kbit:
+- Convert AVI video to MP4. AAC Audio @ 128kbit, h264 Video @ 1250kbit (Gives a predictable file size at lower quality):
 
-`ffmpeg -i {{input_video}}.avi -acodec libfaac -ab 128k -vcodec mpeg4 -b 1250K {{output_video}}.mp4`
+`ffmpeg -i {{input_video}}.avi -codec:audio aac -b:audio 128k -codec:video libx264 -b:video 1250K {{output_video}}.mp4`
+
+- Convert AVI video to MP4. AAC Audio @ 128kbit, h264 Video @ CRF 23, Preset of very slow (Good trade off with file size/speed/quality), also allow faststart by moving the MOOV atom to the start of the file after processing:
```

Comment:
The following 2 examples seems to be an advanced usage of `ffmpeg` to me. Please note that a tldr page is mostly aimed at a beginner to get familiar with the basics of the command. For an exhaustive list, the man page is always there. Also, this takes the no. of examples to 7 which we don't usually go to unless absolutely necessary.

Do you think these examples are required for someone just starting to know the ffmpeg command ? Let me know what you think,

Reply:
Seems like a reasonable ask of a quick summary guide. Although I would suggest that speciting a bitrate for the video us almost never the correct option and suspect that the better "default" for a new user would be to specify a crf and let ffmpeg figure out the but rates.

The remuxing example is actually probaby the most common thing that is asked about on stack exchange/mailing lists that everyone defaults to the wrong action. If a user uses `ffmpeg -I {{video}}.mkv {{video}}.mp4` then ffmpeg will re-encode the whole thing using what it guesses are the best codecs for the format. If you specify to copy then the remux takes seconds and achieves what 99% of users are actually trying to do.

I'll update this pull with a smaller, simplified list of examples of just the bits I think are the most commonly asked questions just from what I've seen, and their most straightforward answers.